src/QsCompiler/QirGeneration/RewriteSteps.cs

```diff
@@ -87,10 +88,13 @@ public bool Transformation(QsCompilation compilation, out QsCompilation transfor
             generator.Emit(bcFile, emitBitcode: true);
             PerformanceTracking.TaskEnd(PerformanceTracking.Task.BitcodeGeneration);
 
-            // create the human readable version as well
-            var sourceOutputFolder = this.AssemblyConstants.TryGetValue(ReservedKeywords.AssemblyConstants.QirOutputPath, out path) && !string.IsNullOrWhiteSpace(path) ? path : "qir";
-            var llvmSourceFile = CompilationLoader.GeneratedFile(targetFile, Path.GetFullPath(sourceOutputFolder), ".ll", "");
-            generator.Emit(llvmSourceFile, emitBitcode: false);
+            if (this.AssemblyConstants.TryGetValue(ReservedKeywords.AssemblyConstants.QirOutputPath, out path) && !string.IsNullOrWhiteSpace(path))
```

Comment:
The behavior here is changing to no longer default to `qir`. If this is the desired behavior, the associated test code should probably have a different value than the old default in order to verify changed values.

Reply:
the fact that the current setup output something even when qiroutputpath was not found is an implementation detail that so far nobody could have relied on (unless they were compiling the command line compiler from source). It is hence fine to change that behavior to not outputting anything unless qiroutputpath is specified. The tests indeed were updated and now require that we set this value. Does that address this comment?